package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-relay-mutation",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
maybe we do the version bump on `master`?

Reply:
Gonna be honest I have no idea what is happening here...maybe this is the current version but didn't push or branched to early?

Reply:
i suppose it doesn't really matter